redbot/cogs/audio/audio.py

```diff
@@ -302,6 +303,29 @@ def _restart_connect(self):
         await self.config.guild(ctx.guild).emptydc_timer.set(seconds)
         await self.config.guild(ctx.guild).emptydc_enabled.set(enabled)
 
+    @audioset.command()
+    @checks.mod_or_permissions(administrator=True)
+    async def maxlength(self, ctx, seconds):
+        """Max length of a track to queue in seconds. 0 to disable.
+
+        Accepts seconds or a value formatted like 00:00:00 (h:m:s) or 00:00 (m:s).
```

Comment:
`(h:m:s)` should be `(hh:mm:ss)` or `(h\\:m\\:s)` otherwise discord renders the `:m:` as an emoji. Alternatively surround the time markers with backticks.